web/layout_attributes/nesting_flex_layouts/my_element.html

```diff
@@ -1,26 +1,26 @@
 <link rel="import" href="packages/polymer/polymer.html">
```

Comment:
I did this is only for examples without Dart code (without an `index.dart` file)